legend-engine-xts-graphQL/legend-engine-xt-graphQL-query/src/main/java/org/finos/legend/engine/query/graphQL/api/cache/GraphQLProdCacheKey.java

```diff
@@ -25,16 +25,38 @@ public class GraphQLProdCacheKey implements GraphQLCacheKey
     private String runtimePath;
     private String queryClassPath;
     private String query;
+    private String dataspacePath;
+    private String executionContext;
 
-    public GraphQLProdCacheKey(String groupID, String artifactId, String versionId, String mappingPath, String runtimePath, String queryClassPath, String query)
+    GraphQLProdCacheKey()
```

Comment:
Should this be private if you intend it to be built with below static methods

Reply:
Function was removed